README.md

```diff
@@ -829,6 +846,8 @@ colord("hwb(210 0% 60% / 50%)").toHex(); // "#00336680"
 
 Adds support of [CIE LAB](https://en.wikipedia.org/wiki/CIELAB_color_space) color model. The conversion logic is ported from [CSS Color Module Level 4 Specification](https://www.w3.org/TR/css-color-4/#color-conversion-code).
 
+Also plugin provides `.delta` method for perceived color difference calculations.
```

Comment:
Please add a simple example of `.delta` function to the plugin section (2 lines is enough).
And also update the plugin size (check it via `npm run size`)